AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_DisplayMapperAdded.py

```diff
@@ -129,52 +191,263 @@ def AtomEditorComponents_DisplayMapper_AddedToEntity():
+            display_mapper_component.set_component_property_value(
+                AtomComponentProperties.display_mapper('Max Point (luminance)'), value=4000.0)
+            Report.result(Tests.luminance_level_max, Math_IsClose(display_mapper_component.get_component_property_value(
+                AtomComponentProperties.display_mapper('Max Point (luminance)')), 4000.0, TOLERANCE))
+
+            # set a low value
+            display_mapper_component.set_component_property_value(
+                AtomComponentProperties.display_mapper('Max Point (luminance)'), value=0.002)
+            Report.result(Tests.luminance_level_max, Math_IsClose(display_mapper_component.get_component_property_value(
+                AtomComponentProperties.display_mapper('Max Point (luminance)')), 0.002, TOLERANCE))
+
+            # restore the default since this impacts the range of 'Mid Point (luminance)'
+            display_mapper_component.set_component_property_value(
+                AtomComponentProperties.display_mapper('Max Point (luminance)'), value=1005.7191162)
+
+            # 17. Set 'Surround Gamma' to high and low value
+            # set a high value
+            display_mapper_component.set_component_property_value(
+                AtomComponentProperties.display_mapper('Surround Gamma'), value=1.2)
+            Report.result(Tests.surround_gamma, Math_IsClose(display_mapper_component.get_component_property_value(
+                AtomComponentProperties.display_mapper('Surround Gamma')), 1.2, TOLERANCE))
+
+            # set a low value
+            display_mapper_component.set_component_property_value(
+                AtomComponentProperties.display_mapper('Surround Gamma'), value=0.6)
+            Report.result(Tests.surround_gamma, Math_IsClose(display_mapper_component.get_component_property_value(
+                AtomComponentProperties.display_mapper('Surround Gamma')), 0.6, TOLERANCE))
+
+            # 18. Set 'Gamma' to high and low value
+            # set a high value
+            display_mapper_component.set_component_property_value(
+                AtomComponentProperties.display_mapper('Gamma'), value=4.0)
+            Report.result(Tests.gamma, Math_IsClose(display_mapper_component.get_component_property_value(
+                AtomComponentProperties.display_mapper('Gamma')), 4.0, TOLERANCE))
+
+            # set a low value
+            display_mapper_component.set_component_property_value(
+                AtomComponentProperties.display_mapper('Gamma'), value=0.2)
+            Report.result(Tests.gamma, Math_IsClose(display_mapper_component.get_component_property_value(
+                AtomComponentProperties.display_mapper('Gamma')), 0.2, TOLERANCE))
+
+            # 19. Enter/Exit game mode.
+            TestHelper.enter_game_mode(Tests.enter_game_mode)
+            general.idle_wait_frames(1)
+            TestHelper.exit_game_mode(Tests.exit_game_mode)
+
+            display_mapper_component.set_component_property_value(
```

Comment:
nit: Same as above. Should there be a comment explaining why Enable LDR color grading LUT is being set to false?